### PR TITLE
Add quality metrics.

### PR DESCRIPTION
**Pull request by nekhtiari**

### CODE REVIEW

File: utils/evaluation.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    )
+
+    model.compile(
+        optimizer=nadam, loss="mean_absolute_error", metrics=["mean_squared_error"]
+    )
+    print("Model compiled.")
+    model.count_params()
+
+    # model = keras.models.load_model(MODEL_PATH +"s2_030_lr_1e-05.hdf5")
+    model.load_weights(MODEL_PATH +"s2_032_lr_1e-04.hdf5")
```
Comment (rodrigoalmeida94) [inline utils/evaluation.py]: I guess we can replace this whole part with `keras.models.load_model` Would be good also if the path to the model was an argument so we can change that easily without touching code

File: utils/evaluation.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            org_img_path = os.path.join(path_to_patches, dset + "/no_tiling/data60_gt.npy")
+            pred_img_path = os.path.join(path_to_patches, dset +  "/no_tiling/data60_predicted.npy")
+        else:
+            org_img_path = os.path.join(path_to_patches, dset + "/no_tiling/data20_gt.npy")
+            pred_img_path = os.path.join(path_to_patches, dset + "/no_tiling/data20_predicted.npy")
+
+        predict_downsampled_img(path, folder, dset, border, final_name)
+        eval_value = evaluation(org_img_path, pred_img_path, metric)
+        gt_sr.append(eval_value)
+    metric_dict["GT_SR"] = sum(gt_sr) / len(gt_sr)
```
Comment (rodrigoalmeida94) [inline utils/evaluation.py]: This i suppose is the mean? We're not saving each individual image metrics?

**Review by rodrigoalmeida94 — COMMENTED** [review]
Cool! A couple of notes.

File: utils/evaluation.py (hunk 1 above)
Comment (nekhtiari) [reply utils/evaluation.py]: Yes but for now I could only use this.

Comment (nekhtiari) [reply utils/evaluation.py]: somehow with keras.models.load_model was failing, but it was the old weight, with the new one I have not tried.

File: utils/evaluation.py (hunk 2 above)
Comment (nekhtiari) [reply utils/evaluation.py]: Exactly, it is the mean of all images.

**Pull request merged**